modules/aws_base/module.yaml

```diff
@@ -0,0 +1,71 @@
+name: aws-base
+clouds:
+  - aws
+input_schema: {
+    "$id": "https://app.runx.dev/modules/aws-base",
```

Comment:
The previous id was of the form
`  "$id": "http://localhost:7800/modules/aws_base/input_schema.aws-base.json",`
why aren't we using localhost here?